.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Testing
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
```

Comment:
Is it simple to extend this to python `3.7` and `3.9` as well?

Reply:
Yes, I will fix it.

Reply:
https://github.com/EvolutionGym/evogym/pull/25/commits/b0649ff34e3f7f4bbeb499e3f363379b2c2e3c17